lib/services/transaction-manager.js

```diff
@@ -1,19 +1,19 @@
 import Promise from 'bluebird'
 import * as constants from '../constants'
 
+import pkg from '../../package.json'
```

Comment:
Include logger also her instead of building new one

Reply:
https://github.com/commercetools/commercetools-node-variant-reassignment/pull/67/commits/839dfb2441ac25df97cd48dce6f5c23a4bed81d4